pkg/multiplexer/multiplexer.go

```diff
@@ -156,6 +157,9 @@ func (cm *ClientMultiplexer) GetEvents(ctx context.Context, allowList, denyList
 				default:
 				}
 				res, err := stream.Recv()
+				if err != nil {
+					res = proto.Clone(res).(*tetragon.GetEventsResponse)
+				}
```

Comment:
one nit: you could add a little comment with the git commit information since it might be easier to understand.

Reply:
I had the same thought this morning haha :sweat_smile: doing it now